galsim/zernike.py

```diff
@@ -433,6 +438,91 @@ def __init__(self, coef, R_outer=1.0, R_inner=0.0):
         self.R_outer = float(R_outer)
         self.R_inner = float(R_inner)
 
+    def __add__(self, rhs):
```

Comment:
Should add doc strings for these arithmetic operations.  We'll want to list them in the Sphinx docs for the class.